layouts/partials/posts/math.html

```diff
@@ -6,10 +6,7 @@
     MathJax = {
       tex: {
         inlineMath: [
-          ['$', '$']
-        ],
-        displayMath: [
-          ['$$', '$$']
+          ['$', '$'], ['\\(', '\\)']
```

Comment:
I think we still need `displayMath` here. Shouldn't it be something like this here:
```
    inlineMath: [
      ['$', '$'], ['\\(', '\\)']
    ],
    displayMath: [
      ['$$', '$$'], ['\\[', '\\]']
    ],
```

Reply:
The default option has treated `$$` and `\\[` as delimiters, so we don't need to override `displayMath`. Remove it should be fine.

Reply:
Ok. Undestood.